data/strings/strings.txt

```diff
@@ -9843,6 +9843,85 @@
     zh-Hans = 下一页
     zh-Hant = 下一頁
 
+  [editor_time_from]
+    comment = Tab title in the time picker
```

Comment:
```suggestion
    comment = Tab title in the Edit Opening Hours time picker
```

Reply:
Please add a similar comment to the To field. It will help translators with the context.

Reply:
Done.